### PR TITLE
version bump 1.3.4

### PR DESCRIPTION
**Pull request by abueg**
gfastats version bump from 1.3.0 to 1.3.3

### CODE REVIEW

File: tools/gfastats/macros.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">1.3.0</token>
+    <token name="@TOOL_VERSION@">1.3.3</token>
```
Comment (bgruening) [inline tools/gfastats/macros.xml:2]: we the conda package is not yet available

Comment (abueg) [reply tools/gfastats/macros.xml:2]: derp, sorry about that... why is bioconda trying to get a version that is not out yet, though: https://github.com/bioconda/bioconda-recipes/pull/36290

Comment (bgruening) [reply tools/gfastats/macros.xml:2]: I think Giulio has played with the builds in the last days.

Comment (abueg) [reply tools/gfastats/macros.xml:2]: ah nevermind, it is a tag, I just didn't see it in the main releases tab, my bad: https://github.com/vgl-hub/gfastats/releases/tag/v1.3.4

**Comment by abueg** [comment]
will update the release number tomorrow so bioconda has time to update!

**Comment by bgruening** [comment]
@abueg the test data needs to be updated, it has changed significantly.

**Review by bgruening — APPROVED** [review]

**Pull request merged**